apps/crn-frontend/src/shared-research/ResearchOutput.tsx

```diff
@@ -17,13 +20,25 @@ const ResearchOutput: React.FC = () => {
   const { path } = useRouteMatch();
   const researchOutputData = useResearchOutputById(researchOutputId);
   const backHref = useBackHref() ?? sharedResearch({}).$;
-  const canCreateUpdate = useCanCreateUpdateResearchOutput(
+  const canCreateUpdateTeam = useCanCreateUpdateResearchOutput(
```

Comment:
We are trying to keep Research Output agnostic of being Team or WG. Can you move these two permission functions to either Team or WG output? It might also make sense to move the context up there too